BUILDING.md

```diff
@@ -86,6 +86,10 @@ $ sudo make install
 ```
 
 This should complete without errors, and `sqlitebrowser` should now be launch-able from the command line.
+**Note 3** 
```

Comment:
Can you add an extra empty line above the ```Note 3``` line? Otherwise the layout is slightly broken :wink: